components/enums.py

```diff
@@ -0,0 +1,9 @@
+import enum
```

Comment:
lets write: from enum import Enum
and use "Enum" at line 4

Reply:
Done